### PR TITLE
To-do List: Add functionalities

### PR DESCRIPTION
**Pull request by Adams-Ghub**
- Develop Add task feature
- Develop edit task feature
- Develop delete task feature

### CODE REVIEW

File: src/modules/Functionalities.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      window.location.reload();
+    } else { localStorage.setItem('tasks', JSON.stringify(data)); }
+  });
+};
+
+export const replaceBtn = (find, replace) => {
+  const substitute = document.querySelector(replace);
+  const out = document.querySelector(find);
+  out.classList.add('hide');
+  substitute.classList.remove('hide');
```
Comment (ghost) [inline src/modules/Functionalities.js]: - [ ] Great work on your app but there are some aspects you have not met yet.  Tasks were not being deleted after clicking on the `delete` icon. Kindly ensure that tasks are being deleted as it is an important feature of the app

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @Adams-Ghub , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Tasks can be added and edited👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/Functionalities.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
```
Comment (IngDavidTM) [inline src/modules/Functionalities.js]: - [ ] Your work on the app is commendable :+1:, but as mentioned by the previous reviewer the tasks that are created in the app cannot be deleted even when you click on the delete icon. As deleting tasks is an essential feature of the app, it is recommended that you fix this issue.


https://user-images.githubusercontent.com/105117832/230613052-b6255ce6-c329-428b-87e4-3a461e3145d3.mp4

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @Adams-Ghub 🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# Pull Request Approved ✅ 🎊 🎉 
## Hello @Adams-Ghub 👋 

## Things to Appreciate 
- [ ] Correct Flow ✅ 
- [ ] Clean Code 👍 
- [ ] No Linter Errors 🔥 
- [ ] Descriptive Pull Request and Summary 👌 

### Your project is complete!  There is nothing else to say other than it's time to merge it.

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 ☑️ 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**